cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -220,6 +219,8 @@ struct Stripe {
   bool validate_sync_serial();
   Errata updateHeaderFooter();
   Errata InitializeMeta();
+  void init_dir();
+  Errata Clear(); // clears striped headers and footers
```

Comment:
OK for now, but really we need to have a convention for method names. I think we'll give up and go STL style - lower case, underscore separator.